### PR TITLE
eNikshay: Add management command for removing unnecessary repeater submissions

### PR DESCRIPTION
**Pull request by NoahCarnahan**
@proteusvacuum cc @calellowitz 
As previously discussed (but adding here for posterity), a previous repeat bug resulted in many case updates being submitted, all of which updated the same two case properties. Some cases have over 4000 forms modifying them. These forms slow down UCRs in the domain.
This management command archives all by the last system form that updated the two case properties. One slight downside to this approach is that some of those forms would have been intentional, since we would expect a few of those repeater generated forms on the case. However, archiving these old forms should not affect the case's current properties.

### CODE REVIEW

File: custom/enikshay/management/commands/archive_extra_repeater_forms.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        to_archive = []
+        episode_case_ids = CaseAccessors(domain).get_case_ids_in_domain("episode")
+        for episode_case_id in with_progress_bar(episode_case_ids[:100]):
+            nikshay_to_archive = []
+            dots_99_to_archvie = []
+            case_forms = FormProcessorInterface(domain).get_case_forms(episode_case_id)
+            for form in case_forms:
+                if form.user_id in ("system", "", None) and form.metadata.username == "system":
+                    updates = get_case_updates(form)
+                    update_actions = [update.get_update_action() for update in updates if update.id == episode_case_id]
```
Comment (dimagimon) [inline custom/enikshay/management/commands/archive_extra_repeater_forms.py]: E501 line too long (119 > 115 characters)

File: custom/enikshay/management/commands/archive_extra_repeater_forms.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+                    for action in update_actions:
+                        if isinstance(action, CaseUpdateAction):
+                            if set(action.dynamic_properties.keys()) == {"nikshay_registered", "nikshay_error"}:
+                                nikshay_to_archive.append(form)
+                            elif set(action.dynamic_properties.keys()) == {"dots_99_registered", "dots_99_error"}:
+                                dots_99_to_archvie.append(form)
+
+
+            nikshay_to_archive = nikshay_to_archive[:-1]
```
Comment (dimagimon) [inline custom/enikshay/management/commands/archive_extra_repeater_forms.py]: E303 too many blank lines (2)

File: custom/enikshay/management/commands/archive_extra_repeater_forms.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+            action='store_true',
+            dest='commit',
+            default=False,
+            help="Don't do a dry run, but actually archive the forms",
+        )
+
+    def handle(self, log_file, **options):
+        to_archive = []
+        episode_case_ids = CaseAccessors(domain).get_case_ids_in_domain("episode")
+        for episode_case_id in with_progress_bar(episode_case_ids[:100]):
```
Comment (proteusvacuum) [inline custom/enikshay/management/commands/archive_extra_repeater_forms.py]: why `[:100]`?

Comment (NoahCarnahan) [inline custom/enikshay/management/commands/archive_extra_repeater_forms.py]: ah shit, meant to remove that :P (limited the number when I was testing it)

**Pull request merged**